config/software/nokogiri.rb

```diff
@@ -33,11 +33,11 @@
 
 # nokogiri uses pkg-config, and on a mac that will find the system pkg-config
 # which will find the system pkg-configs which will pull in libicucore from the
-# libxml2 pkg-config spec.  override pkg-configs path here to point into our
-# /opt/chef/embedded pkg-configs.  this should probably be done more generally,
+# libxml2 pkg-config spec.  override pkg-configs libdir here to only point into
+# our /opt/chef/embedded pkg-configs. This should probably be done more generally,
 # in core ominbus-ruby.
 env = {
-  "PKG_CONFIG_PATH" => "#{install_dir}/embedded/lib/pkgconfig",
+  "PKG_CONFIG_LIBDIR" => "#{install_dir}/embedded/lib/pkgconfig",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why are we changing this env var name ?

Reply:
allow me to point out the commit message :)
> PKG_CONFIG_PATH extends the default system path, which could lead to the system
provided .pc files to be used.
PKG_CONFIG_LIBDIR overrides the defaults entirely, causing only the .pc files in
the path provided as value for that environment variable to be probed